src/HttpFoundation/Response/NoContent.php

```diff
@@ -1,19 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace CultuurNet\UDB3\HttpFoundation\Response;
-
-use Symfony\Component\HttpFoundation\Response;
-
-/**
- * @deprecated
- *   Use CultuurNet\UDB3\Http\Response\NoContentResponse instead.
- */
-class NoContent extends Response
```

Comment:
Thanks for also cleaning this up! 👍

Reply:
ah, looking at this, I could have avoided your comments on the part 2 PR :)